### PR TITLE
fix(browserless): attach page target for CDP v2

### PR DESCRIPTION
**Pull request by chaliy**
## What
Attach Browserless CDP connections to a page target before issuing `Page.*`, `Runtime.*`, or `Input.*` commands, and expose the attached target/session ids for live smoke coverage.
Update the Browserless live tests to assert the attach handshake and to use a reconnect TTL that fits the current live token limits.
Document the Browserless v2 page-target attach requirement in the integration spec.

## Why
Browserless `/chromium` now returns a browser-level CDP session, so page-scoped commands fail on the root session with `Page.enable wasn't found`.
Fixes EVE-379.

## How
On every `CdpSession::connect`, inspect browser targets, attach to an existing `page` target when present, or create `about:blank` and attach if none exist.
Route `Page.*`, `Runtime.*`, and `Input.*` envelopes through the attached `sessionId`; keep `Target.*` and `Browserless.*` on the browser root session.
Add regression tests for both attach flows plus a live Browserless smoke that asserts a non-empty `sessionId` after connect.
Validated with `cargo test -p everruns-integrations-browserless`, `doppler run -- cargo test -p everruns-integrations-browserless --features browserless-live-tests --test live_api -- --test-threads=1`, and `just pre-push`.

## Risk
- Low
- Could attach to the wrong page target if Browserless exposes multiple tabs and the intended page is not the first non-placeholder page. Unit and live reconnect coverage exercise the common single-page session flow.

## Security
- Threat categories reviewed: `TM-TOOL`
- Findings and resolutions:
  - Reviewed the new CDP envelope routing and target selection path for tool-execution risk. No new auth, secret, or storage surface was introduced.
  - Browser-wide commands stay sessionless; page-scoped commands now route only after explicit `Target.attachToTarget`, which removes the root-session method dispatch failure without widening permissions.

## Checklist
- [x] Tests added or updated
- [x] Backward compatibility considered
- [x] Security review performed against relevant threat model categories
- [x] All review comments addressed (code change or written reasoning)

### CODE REVIEW

File: integrations/browserless/src/cdp.rs [hunk 1]
```diff
@@ -165,6 +206,64 @@ impl CdpSession {
...
+            )
+            .await?;
+
+        let session_id = result
+            .get("sessionId")
+            .and_then(|value| value.as_str())
+            .ok_or_else(|| "Target.attachToTarget returned no sessionId".to_string())?;
+
+        self.page_target_id = target_id;
+        self.page_session_id = session_id.to_string();
```
[the remark this answers is not in the document]
Comment (chaliy) [reply integrations/browserless/src/cdp.rs:231]: Checked against the current branch and reran the Browserless crate validation.  is still available after the  call here, so there is no use-after-move in the compiled code path; no code change was needed.

[the remark this answers is not in the document]
Comment (chaliy) [reply integrations/browserless/src/cdp.rs:231]: Checked against the current branch and reran the Browserless crate validation. `target_id` is still available after the `json!` call here, so there is no use-after-move in the compiled code path; no code change was needed.
